conf/modules.config

```diff
@@ -493,15 +493,15 @@ process {
         publishDir = [
             path: { "${params.outdir}/itsx" },
             mode: params.publish_dir_mode,
-            pattern: "ASV_*"
+            pattern: "mergepairs_consensus_*"
```

Comment:
I think that was too much :D

Reply:
Sorry for this, a find/replace that turned wrong 🥲